modules/ROOT/pages/constraints/examples.adoc

```diff
@@ -472,6 +472,12 @@ Assuming a constraint with the name `sequels` already exists:
 (no changes, no records)
 ----
 
+.Notification
```

Comment:
For the docs team review: I'm not sure how we want to represent this and if/how we want to add it to the docs testing

Reply:
thanks, i'll ask in our daily stand-up tomorrow

Reply:
I think the choice you've made is good. There's no plan to test notifications as of now, also given how few of them we have in docs (if any, beyond this addition).